release_notes/unreleased.md

```diff
@@ -1 +1,6 @@
 **Unreleased**
+
+**Version 2.2.0 - Released February 08, 2022**
+
+* Added support for 'list files' action [PAPP-7599]
+* Added support for 'get file' action [PAPP-7599]
```

Comment:
Please refer to other unreleased.md and correct the format. merge both points into one